part-2/getting-started.md

```diff
@@ -2,20 +2,17 @@
 
 ## Software and Hardware Requirements
 
-| OS         | Windows 7®, Windows 8®, Windows 10®, Windows 11®, Windows Server 2008 R2®, Windows Server 2012 R2®, Windows Server 2016®, Windows Server 2019® |
-| ---------- | -------------------------------------- |
-| Processor  | minimum 1 GHz (the faster, the better) |
-| Memory     | 2 GB RAM, 4+ GB recommended            |
-| Disk space | minimum 2 GB                           |
-||              **Optional software**                 |
-| R®         | version 3.6 or higher - 64bit        |
+| OS | Windows 7®, Windows 8®, Windows 10®, Windows 11®, Windows Server 2008 R2®, Windows Server 2012 R2®, Windows Server 2016®, Windows Server 2019® |
+|-------------------|-----------------------------------------------------|
+| Processor | minimum 1 GHz (the faster, the better) |
+| Memory | 2 GB RAM, 4+ GB recommended |
+| Disk space | minimum 2 GB |
+|  | **Optional software** |
+| R® | version 4.1 or higher - 64bit |
 
+The information provided above refers to the core components of the Open Systems Pharmacology Suite, including PK-Sim® MoBi®. Both PK-Sim® and MoBi® can be installed as stand-alone software packages to reduce the disk space required.
 
-Information provided above refers to the core components of the Open Systems Pharmacology Suite including PK-Sim®, MoBi®. Both, PK-Sim® and MoBi® can be installed as stand-alone software packages to reduce the disk space required.
-
-{% hint style="warning" %}
-The Open Systems Pharmacology Suite includes interfaces to MS Excel® and R (version 3.5 or 3.6, 64 bit). These are separate programs that are not available within the Open Systems Pharmacology Suite. You need to have these programs installed in order to use their interfaces!
-{% endhint %}
+{% hint style="warning" %} The Open Systems Pharmacology Suite includes interfaces to MS Excel® and R. These are separate programs that are not available within the Open Systems Pharmacology Suite. You need to have these programs installed to use their interfaces! {% endhint %}
```

Comment:
here and below: the gitbook instructions `{% hint ... %}` and `{% endhint %}` are placed inline and not on a separate lines anymore.
@msevestre Is it OK or will it create problems when rendering with GitBook?

Reply:
Reverted to previous behavior.

Reply:
Would be good to know if the inline style is supported @msevestre as Positron editor (the new R IDE from Posit, based on VSC) automatically converts it to this style.